js/plugins/code/base-text-element.js

```
@@ -1774,21 +2350,55 @@ class BaseTextElement extends HTMLElement {
+            // No HTML data - check if it's plain text that looks like markdown
             const text = clipboardData.getData('text') || clipboardData.getData('text/plain');
             if (text) {
                 event.preventDefault();
+
+                // Check if text looks like markdown
+                if (this.isMarkdownText(text)) {
+                    console.log('Detected markdown text, parsing...');
+                    const parsedElements = this.parseMarkdownText(text);
+
+                    if (parsedElements.length > 0) {
+                        var inx = 0;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'inx'. Consider to delete it or rename it.

'inx' is defined here:


(lint/suspicious/noRedeclare)